changelog/23534.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
I wonder if this deserves to be a feature? Customers have been asking for it, so it'd make it show up more prominently